lib/chef-dk/policyfile_services/update_attributes.rb

```diff
@@ -97,6 +101,10 @@ def assert_policy_and_lock_present!
         end
       end
 
+      def prepare_constraints
+        Policyfile::LockApplier.new(policyfile_lock, policyfile_compiler).
+          apply!
```

Comment:
This is a weird line wrap.

Reply:
there's a dot in the previous line. i was trying to do fancy builder stuff